scripts/tables_2023.R

```diff
@@ -109,8 +109,8 @@ hab_fish_indiv <- full_join(
     T ~ NA_character_
   ),
   life_stage = case_when(
-    species_code %in% c('L', 'SU', 'LSU') ~ NA_character_,
-    T ~ life_stage
+    stringr::str_detect(species_code, 'L|SU|LSU') ~ NA_character_,
```

Comment:
ha! %in% is all good as it is base R.  its not from the data.table package like %like% is

Reply:
in fact those two calls do different things that prob don't matter here but ...

%in% is exact match where as detect just needs those characters to be present (ex. LNC will result in NA_character)

Reply:
ooooo good catch thank you. needs to change here then as well https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/blob/main/scripts/01_prep_inputs/0100-extract-inputs.R#L664C2-L670C21